pkg/sql/opt/optbuilder/misc_statements.go

```diff
@@ -108,10 +108,6 @@ func (b *Builder) buildCancelSessions(n *tree.CancelSessions, inScope *scope) (o
 func (b *Builder) buildControlSchedules(
 	n *tree.ControlSchedules, inScope *scope,
 ) (outScope *scope) {
-	if err := b.catalog.RequireAdminRole(b.ctx, n.StatementTag()); err != nil {
```

Comment:
confirm owner is checked elsewhere?

Reply:
Owner and admin are now checked in `control_schedules.go` over here - https://github.com/cockroachdb/cockroach/pull/87287/files#diff-2a63f9cb5b50b6a09ff56a751c7bb34269c8e8832c57592f41f0cdbd12d27ddeR140

Reply:
oh, of course, it's the same command :) thanks!